playground/css/vite.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
       '=': __dirname,
       spacefolder: __dirname + '/folder with space',
       '#alias': __dirname + '/aliased/foo.css',
+      '#alias?inline': __dirname + '/aliased/foo.css?inline',
```

Comment:
`resolve.alias` doesn't handle query. So we need to add this. (It's not needed if `#alias` maps to `__dirname + '/aliased'`)